plugins/kubernetes/db/migrate/20190124171921_bump_limit.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class BumpLimit < ActiveRecord::Migration[5.2]
+  def change
+    change_column :kubernetes_release_docs, :resource_template, :text, limit: 1073741823
```

Comment:
what is this magic number, and why doesn't it match the number in the schema.rb?

Reply:
max mysql column length

Reply:
comment / use a constant?

Reply:
Same question on the numbers, is it safe to set "unlimited" limits?

Reply:
🤷‍♂️ not super safe, but we have been doing it for job output all these years and nothing bad happened

Reply:
comment added